<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>rglab | Research</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le styles -->
    <link href="./utils/css/bootstrap.css" rel="stylesheet">
    <link href="./utils/css/bootstrap-responsive.css" rel="stylesheet">
    <link href="./utils/css/rglab.css" rel="stylesheet">
	
    <!-- Le fav and touch icons -->
    <!-- <link rel="shortcut icon" href="images/favicon.ico">
    <link rel="apple-touch-icon" href="images/apple-touch-icon.png">
    <link rel="apple-touch-icon" sizes="72x72" href="images/apple-touch-icon-72x72.png">
    <link rel="apple-touch-icon" sizes="114x114" href="images/apple-touch-icon-114x114.png"> -->
  </head>
  
  
  
  
  <body data-spy="scroll" data-target=".subnav" data-offset="50">
	  <div class="navbar navbar-inverse navbar-fixed-top">
		  <div class="navbar-inner">
			  <div class="container">
				  <a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
					  <span class="icon-bar"></span>
					  <span class="icon-bar"></span>
					  <span class="icon-bar"></span>
				  </a>
				  <a class="brand" href="./index.html">rglab</a>
				  <div class="nav-collapse">
					  <ul class="nav">
						  <li><a href="./index.html"><i class="icon-home icon-black"></i> Home</a></li>
						  <li class="active"><a href="#"><i class="icon-pencil icon-white"></i> Research</a></li>
						  <li><a href="./publications.html"><i class="icon-book icon-black"></i> Publications</a></li>
						  <li><a href="./software.html"><i class="icon-hdd icon-black"></i> Software</a></li>
						  <li><a href="./members.html"><i class="icon-user icon-black"></i> Members</a></li>
						  <li><a href="./jobs.html"><i class="icon-search icon-black"></i> Jobs</a></li>
						  <li><a href="./contact.html"><i class="icon-envelope icon-black"></i> Contact</a></li>
						  <li><a href="./conferences.html"><i class="icon-calendar icon-black"></i> Conferences</a></li>

					  </ul>
				  </div><!--/.nav-collapse -->		  
			  </div>
		  </div>
	  </div>



		  <!-- Masthead ================================================== -->
		  <header class="jumbotron subhead" id="Members">
		  	  <div class="container">		  
			  <h1><a href="http://www.rglab.org" title="Raphael Gottardo's Research Lab">rglab</a>::research</h1>
			  <p class="lead">Raphael Gottardo's Research Lab</p>
			  	  </div><!-- /container -->			  
		  </header><!-- /header -->

<div class="navbar">
  <div class="navbar-inner">
    <div class="container">    

<form class="navbar-search pull-right" action="http://google.com/search" method="get">
    <input type="hidden" name="q" value="site:rglab.org" />						  
	<input type="text" class="search-query" name="q" placeholder="Search" title="Google search for rglab"/>
</form>
		  </div>
	  </div>
</div>


	  <div class="container">
		  <div class="row">				
			  <div class="span12">
				  <div class="well well-small">									
				    <h3>What do we do?</h3>				  
					Our main research interest is the development of cutting edge statistical methods and software tools for the analysis of high throughput biological data with an emphasis on immunology and vaccine research. Please use the tab navigation below to learn more about specific active research projects in the lab. 
				  </div><!-- /well -->				  
			  </div><!-- /span -->
			  <div class="span12">			 				  
				  <div class="tabbable tabs">
					  <ul class="nav nav-tabs">
						  <li class="active"><a href="#1" data-toggle="tab">Cytometry</a></li>
						  <li><a href="#2" data-toggle="tab">Peptide microarrays</a></li>
						  <li><a href="#3" data-toggle="tab">Single-cell gene expression</a></li>
						  <li><a href="#4" data-toggle="tab">Immunology and vaccine research</a></li>						  
						  <li><a href="#5" data-toggle="tab">Next generation sequencing</a></li>
						  <li><a href="#6" data-toggle="tab">Statistical computing</a></li>						  
						  
					  </ul>
					  <div class="tab-content">
						  <div class="tab-pane active" id="1">
							  <p>We do research on new statistical methods for analysing flow cytometry data, including methods for determining vaccine positivity, automated and semi-automated gating and data analysis pipelines, as well as quality assurance and quality control.</p>
							  </div>
					
						  <div class="tab-pane" id="2">
							  <p>Our group researches advanced statistical modeling techniques to improve the power to detect antibody epitopes using peptide microarray experiments. We aim to develop flexible, robust methodology that shares relevant information between subjects while accommodating aspects of experiments that are unique to immunological applications. We are also developing a unified software environment to streamline data analysis from preprocessing up through detection and visualization. Our methodology is currently being applied to many vaccine trials including the human and non-human primate studies. 							  </p>
							  
								<p>The two figures below show an application of our analysis and visualization pipeline applied to the RV144 pilot microarray data published in <em>AIDS Research and Human Retroviruses</em>.</p><br />
							  	<ul class="thumbnails">
							  	  <li class="span6">
							  	    <a href="./utils/img/V2-RV144.png" class="thumbnail" id="pep1" title="RV144 V2 loop aggregate response">
								  <img src="./utils/img/V2-RV144.png" alt="V2_loop_aggregate_response">
							  	    </a>
							  	      <h4>RV144 V2 loop aggregate response</h4>
							  	      <p>We can observe a strong V2 response in the vaccine group and no response in the placebo group. </p>
							  	  </li>
							  	    <li class="span6">
							  	      <a href="./utils/img/V2-RV144-subtype.png" class="thumbnail" id="pep2" title="RV144 V2 loop subtype specific response">
							  	  <img src="./utils/img/V2-RV144-subtype.png" alt="V2_loop_subtype_specific_response">
							  	      </a>
							  	        <h4>RV144 V2 loop subtype specific responses</h4>
							  	        <p>The strongest response is observed in CRF01 whereas B is the smallest.</p>
							  	    </li>							  	  
							  	</ul>
						  </div>
						  <div class="tab-pane" id="3">							  
						  	  <ul class="thumbnails">
						  	  <li class="span6">
						  	   <a href="./utils/img/Fluidigm.png" class="thumbnail" id="cyto1" title="Single-cell gene expression in tetramer-sorted cell populations">
						  		<img src="./utils/img/Fluidigm.png" alt="Single_Cell_Gene_Expression">
						  		</a>
						  		<h4>Single-Cell Gene Expression and Positivity</h4>
						  		<p>Subject-specific differences as well as stimulation-specific common patterns in single-cell gene expression are observed between differentially stimulated, tetramer-sorted cell subpopulations. The data was analyzed using our new MIMOSA (Mixture Models for Single-Cell Assays) tool.</p>
						  		</li>
						  		</ul>						  
						  	  </div>
						  <div class="tab-pane" id="4">
						  	  <p>We collaborate with bench scientists and clinicians throughout multiple networks including the HIV Vaccine Trial Network, the Microbicide Trial Network, the Immune Tolerance Network and the Human Immunology Project Consortium.</p>
						  </div>
						  
						  <div class="tab-pane" id="5">
							  <p>For the past decade, microarrays have been the platform of choice for various high-throughput biological studies. In fact, our group has significantly contributed to the development of statistical methods for microarray data.
							 Recent improvements in the efficiency, quality, and cost of genome-wide sequencing are 
							 prompting biologists to rapidly abandon microarrays in favor of so-called next-generation sequencers. Next-generation sequencing technologies, capable of producing tens of 
							 millions of sequence reads during each instrument run, have already been applied for 
							 studying genome-wide transcription levels (RNA-Seq), transcription factor binding 
							 sites and nucleosome positioning (ChIP-Seq), and DNA methylation status. While 
							 sequencing-based studies have been promoted as overcoming longstanding limitations 
							 of microarray-based studies, these new biotechnologies raise similar as well as 
							 novel statistical and computational challenges. These sequencing technologies generate 
							 massive data sets (much larger than with microarrays), and there is a 
							 pressing need for statistical and computational tools to make sense of these data.
							  
							  To this end, our group has developed several methods for the analysis of ChIP-Seq and MNase-Seq data. The Figures below illustrate our analysis pipeline using 
					<a href="http://www.bioconductor.org/packages/devel/bioc/html/PICS.html" title="Probabilistic Inference for ChIP-Seq" target="_blank">PICS</a>,
					 <a title="de novo motif discovery" href="http://www.bioconductor.org/packages/devel/bioc/html/rGADEM.html" target="_blank">rGADEM</a>,
					 <a title="Motif Identification and Validation" href="http://www.bioconductor.org/packages/devel/bioc/html/MotIV.html" target="_blank">MotIV</a> and
					 <a href="http://www.bioconductor.org/packages/devel/bioc/html/PING.html" title="Probabilistic Inference for Nucleosome Positioning" target="_blank">PING</a>.
						</p><br/>
						<ul class="thumbnails">
						<li class="span6">
						 <a href="./utils/img/RG_SW_bwfigure2.png" class="thumbnail" id="epi1" title="ChIP-Seq analysis with PICS">
							<img src="./utils/img/RG_SW_bwfigure2.png" alt="PICS_example">
							</a>
							<h4>PICS increases spatial resolution of binding event detection</h4>
							<p>Transcription factor binding sites identified by PICS, iSeq, BayesPeak and MACS. PICS identifies two binding events in this region while other three methods identify these two events as one broad enriched region.</p>
							</li>
							<li class="span6">
							 <a href="./utils/img/RG_SW_bwfigure4.png" class="thumbnail" id="epi2" title="Nucleosome positioning analysis with PING">
								<img src="./utils/img/RG_SW_bwfigure4.png" alt="Nucleosome positioning analysis with PING">
								</a>
								<h4>PING identifies weakly positioned nucleosomes.</h4>
								<p>Expected Nucleosome positioning in Chr1 for 149000-153000 genomic region using PING and TpF. PING can identify nucleosome positioning with weak signal compared to TpF.</p>
								</li>							
							</ul>						  												
						  </div>
						  
						  <div class="tab-pane" id="6">
							  <p></p>
						  </div>
					  </div>
				  </div> <!-- /tabbable -->			  
			  </div><!-- /span -->
		  </div><!-- /row -->
	  </div><!-- /container -->		  
		
	  <div class="container">
		  <div class="row">																
			  <div class="span12">
				  <!-- Footer ================================================== -->
				  <footer class="footer">
					  <p class="pull-right"><a href="#">Back to top</a></p>
					  <p class="pull-left">
						  <a href="https://twitter.com/raphg" class="twitter-follow-button" data-width="225px" data-link-color="#0069D6" data-show-count="true">Follow @raphg</a>
						  <a href="https://twitter.com/share" class="twitter-share-button" data-url="http://twitter.github.com/raphg/" data-count="horizontal" data-via="raphg" >Tweet</a></p>
						  <br /><br />
						  <p class="pull-left"> &copy; 2012 <a href="#">rglab</a>. All right reserved.</p>
						  <br />
						  <p class="pull-left">Made with <a href="http://twitter.github.com/bootstrap">Bootstrap</a>.</p>
						  
					  </footer>
			  </div><!--span12-->
		  </div>
	  </div><!-- /container -->
    
    <!-- Le javascript -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script type="text/javascript" src="http://platform.twitter.com/widgets.js"></script>	
    <script src="./utils/js/jquery.js"></script>
    <script src="./utils/js/bootstrap.js"></script>
	<script type="text/javascript" src="./utils/js/jquery.tweetable.js"></script>

	  <!-- Add fancyBox -->
	<link rel="stylesheet" href="./utils/fancybox/source/jquery.fancybox.css?v=2.1.0" type="text/css" media="screen" />
	<script type="text/javascript" src="./utils/fancybox/source/jquery.fancybox.pack.js?v=2.1.0"></script>

	
	<script type="text/javascript">
	$(document).ready(function() {
	$("#cyto1").fancybox({
	helpers : {
		title : {
			type : 'over'
		}
	}
	});});
	
	$(document).ready(function() {
	$("#pep1").fancybox({
    	helpers : {
    		title : {
    			type : 'over'
    		}
    	}
    });	});
    
    $(document).ready(function() {
    $("#pep2").fancybox({
    	helpers : {
    		title : {
    			type : 'over'
    		}
    	}
    });	});

$(document).ready(function() {
$("#epi1").fancybox({
	helpers : {
		title : {
			type : 'over'
		}
	}
});	});

$(document).ready(function() {
$("#epi2").fancybox({
	helpers : {
		title : {
			type : 'over'
		}
	}
});	});
    
	</script>	
	
  </body>
</html>
