<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>rglab | Research</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le styles -->
    <link href="./utils/css/bootstrap.css" rel="stylesheet">
    <link href="./utils/css/bootstrap-responsive.css" rel="stylesheet">
    <link href="./utils/css/rglab.css" rel="stylesheet">
	
    <!-- Le fav and touch icons -->
    <!-- <link rel="shortcut icon" href="images/favicon.ico">
    <link rel="apple-touch-icon" href="images/apple-touch-icon.png">
    <link rel="apple-touch-icon" sizes="72x72" href="images/apple-touch-icon-72x72.png">
    <link rel="apple-touch-icon" sizes="114x114" href="images/apple-touch-icon-114x114.png"> -->
  </head>
  
  
  
  
  <body data-spy="scroll" data-target=".subnav" data-offset="50">
	  <div class="navbar navbar-inverse navbar-fixed-top">
		  <div class="navbar-inner">
			  <div class="container">
				  <a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
					  <span class="icon-bar"></span>
					  <span class="icon-bar"></span>
					  <span class="icon-bar"></span>
				  </a>
				  <a class="brand" href="./index.html">rglab</a>
				  <div class="nav-collapse">
					  <ul class="nav">
						  <li><a href="./index.html"><i class="icon-home icon-black"></i> Home</a></li>
						  <li class="active"><a href="#"><i class="icon-pencil icon-white"></i> Research</a></li>
						  <li><a href="./publications.html"><i class="icon-book icon-black"></i> Publications</a></li>
						  <li><a href="./software.html"><i class="icon-hdd icon-black"></i> Software</a></li>
						  <li><a href="./members.html"><i class="icon-user icon-black"></i> Members</a></li>
						  <li><a href="./jobs.html"><i class="icon-search icon-black"></i> Jobs</a></li>
						  <li><a href="./contact.html"><i class="icon-envelope icon-black"></i> Contact</a></li>
						  <li><a href="./conferences.html"><i class="icon-calendar icon-black"></i> Conferences</a></li>

					  </ul>
				  </div><!--/.nav-collapse -->		  
			  </div>
		  </div>
	  </div>



		  <!-- Masthead ================================================== -->
		  <header class="jumbotron subhead" id="Members">
		  	  <div class="container">		  
			  <h1><a href="http://www.rglab.org" title="Raphael Gottardo's Research Lab">rglab</a>::research</h1>
			  <p class="lead">Raphael Gottardo's Research Lab</p>
			  	  </div><!-- /container -->			  
		  </header><!-- /header -->

<div class="navbar">
  <div class="navbar-inner">
    <div class="container">    

<form class="navbar-search pull-right" action="http://google.com/search" method="get">
    <input type="hidden" name="q" value="site:rglab.org" />						  
	<input type="text" class="search-query" name="q" placeholder="Search" title="Google search for rglab"/>
</form>
		  </div>
	  </div>
</div>


	  <div class="container">
		  <div class="row">				
			  <div class="span12">
				  <div class="well well-small">									
				    <h3>What do we do?</h3>				  
					Our main research interest is the development of cutting edge statistical methods and software tools for the analysis of high throughput biological data with an emphasis on immunology and vaccine research. Please use the tab navigation below to learn more about specific active research projects in the lab. 
				  </div><!-- /well -->				  
			  </div><!-- /span -->
			  <div class="span12">			 				  
				  <div class="tabbable tabs">
					  <ul class="nav nav-tabs">
						  <li class="active"><a href="#1" data-toggle="tab">Cytometry</a></li>
						  <li><a href="#2" data-toggle="tab">Peptide microarrays</a></li>
						  <li><a href="#3" data-toggle="tab">Single-cell expression</a></li>
						  <li><a href="#4" data-toggle="tab">Immunology/Vaccinology</a></li>						  
						  <li><a href="#5" data-toggle="tab">Epigenomics</a></li>
						  <li><a href="#6" data-toggle="tab">Statistical computing</a></li>						  
						  
					  </ul>
					  <div class="tab-content">
						  <div class="tab-pane active" id="1">
							  <p>We do research on new statistical methods for analyzing flow cytometry data, including methods for determining vaccine positivity, automated and semi-automated gating and data analysis pipelines, as well as quality assurance, quality control, and visualization.</p>
							  <ul class="thumbnails">
								  <li class="span6">
									  <a href="./utils/img/flowViz.png" class="thumbnail" id="flow1" title="flowViz visualization of flow cytometry data conditioning on study variables.">
										  <img src="./utils/img/flowViz.png" alt="flowViz_Visualization">
									  </a>
									  <h4>
										  FlowViz Visualization With Conditioning on Study Variables
									  </h4>
									  <p>
										  We have extended <a href="https://github.com/RGLab/flowViz">flowViz</a> to allow for visualization of large data sets of manually gated flow data with associated study metadata using the standard R formula interface. Data exploration and quality control are greatly simplified with the new API.
									  </p>
								  </li>
								  </ul>
							  </div>
					
						  <div class="tab-pane" id="2">
							  <p>Our group researches advanced statistical modeling techniques to improve the power to detect antibody epitopes using peptide microarray experiments. We aim to develop flexible, robust methodology that shares relevant information between subjects while accommodating aspects of experiments that are unique to immunological applications. We are also developing a unified software environment to streamline data analysis from preprocessing up through detection and visualization. Our methodology is currently being applied to many vaccine trials including  human and non-human primate studies. 							  </p>
							  
								<p>The two figures below show an application of our analysis and visualization pipeline applied to the RV144 pilot microarray data published in <a href="http://online.liebertpub.com/doi/full/10.1089/aid.2012.0103" target="_blank"><em>AIDS Research and Human Retroviruses</em></a>.</p><br />
							  	<ul class="thumbnails">
							  	  <li class="span6">
							  	    <a href="./utils/img/V2-RV144.png" class="thumbnail" id="pep1" title="RV144 V2 loop aggregate response">
								  <img src="./utils/img/V2-RV144.png" alt="V2_loop_aggregate_response">
							  	    </a>
							  	      <h4>RV144 V2 loop aggregate response</h4>
							  	      <p>We can observe a strong V2 response in the vaccine group and no response in the placebo group. </p>
							  	  </li>
							  	    <li class="span6">
							  	      <a href="./utils/img/V2-RV144-subtype.png" class="thumbnail" id="pep2" title="RV144 V2 loop subtype specific response">
							  	  <img src="./utils/img/V2-RV144-subtype.png" alt="V2_loop_subtype_specific_response">
							  	      </a>
							  	        <h4>RV144 V2 loop subtype specific responses</h4>
							  	        <p>The strongest response is observed in CRF01 whereas B is the smallest.</p>
							  	    </li>							  	  
							  	</ul>
						  </div>
						
						  <div class="tab-pane" id="3">
						  <p>Although cells sorted using well-established surface markers may appear homogeneous, mRNA expression of other genes within these cells can be heterogeneous and could further characterize cell poly-functionality. New technologies based on microfluidic arrays combined with multiplexed polymerase chain reactions (PCR) can now be used to perform thousands of PCRs in a single device, enabling simultaneous, high-throughput gene expression measurements at the single-cell level across hundreds of cells and genes. While classic gene expression microarrays sum the expression from many individual cells, the intrinsic stochastic nature of biochemical processes results in relatively large cell-to-cell gene expression variability.This heterogeneity may carry important information, thus single cell expression data should not be analyzed in the same fashion as cell-population level data. We are currently developing statistical methods and tools for the analysis of single cell gene expression data (<em>e.g.</em> <a href="http://arxiv.org/abs/1208.5809" target="_blank">MIMOSA</a>, <a href="http://arxiv.org/abs/1210.1226" target="_blank">SingleCellAssay</a>).
						  </p>
						  	  <ul class="thumbnails">
						  	  <li class="span6">
						  	   <a href="./utils/img/Fluidigm.png" class="thumbnail" id="cyto1" title="Single-cell gene expression in tetramer-sorted cell populations">
						  		<img src="./utils/img/Fluidigm.png" height="500" width="500" alt="Single_Cell_Gene_Expression">
						  		</a>
						  		<h4>Single-Cell Gene Expression and Positivity</h4>
						  		<p>Subject-specific differences as well as stimulation-specific common patterns in single-cell gene expression are observed between differentially stimulated, tetramer-sorted cell subpopulations. The data was analyzed using our new MIMOSA (Mixture Models for Single-Cell Assays) tool.</p>
						  		</li>
								<li class="span6">
									<a href="./utils/img/concordcomp.png" class="thumbnail" id="cyto2" title="Concordance between single-cell and 100-cell Fluidigm gene expression experiments">
										<img src="./utils/img/concordcomp.png" height="500" width="455" alt="Single_Cell_Hundred_Cell_Concordance">
									</a>
									<h4>Concordance of Single-Cell and Hundred-Cell Gene Expression Experiments</h4>
									<p>The concordance between single-cell gene expression experiments and the average per-cell expression from 100-cell gene expression experiments is shown for three different data sets (columns) when unexpressed genes are treated as missing observations (second row) or as exact zeros (top row). Filtering of outlier values is applied (last row) improving concordance between the experiments. Treating unexpressed genes as zeros rather than missing observations improves the concordance, indicating that these genes are truly not expressed rather than undetectable through this technology.
						  		</ul>						  
						  	  </div>
						  <div class="tab-pane" id="4">
						  	  <p>We collaborate with bench scientists and clinicians throughout multiple networks including the <a href="http://www.hvtn.org" target="_blank">HIV Vaccine Trials Network</a>, the <a href="http://www.mtnstopshiv.org" target="_blank">Microbicide Trials Network</a>, the <a href="http://www.immunetolerance.org">Immune Tolerance Network</a> and the <a href="http://www.immuneprofiling.org" target="_blank">Human Immunology Project Consortium</a>. In particular we develop and implement tools and methods for data processing and analysis of immunological assays used for measuring vaccine immunogenicity (e.g., ICS, multiplex bead arrays), increasingly involving higher dimensional assays (e.g., mass cytometry for assessing high-dimensional functions of single cells), and to provide statistical validation and standardization of assays. Using these tools, our aim is to					  	  						  	  
						  	  use system approaches to predict how the immune system will respond to a particular infection of vaccination. </p>
						  	  Here are some of the applications we are currently working on:
						  	  <dl>
						  	    <dt>Epitope mapping</dt>
								<dd>
									In collaboration with the <a href="http://www.hvtn.org" target="_blank">HVTN</a> we are testing new methods for positivity calls (<a href="http://arxiv.org/abs/1208.5809" target="_blank">MIMOSA</a>) on other assays, including epitope mapping and EliSpot.
								</dd>
						  	    <dd></dd>
						  	    <dt>Immune correlates</dt>						  	    
						  	    <dd>We are currently working on identifying immune response biomarkers that correlate with vaccine efficacy (<em>e.g.</em> protection/infection) for the RV144 and RTS'S trials.
						  	    </dd>
								<dt>Poly-functionality</dt>
								<dd>In collaboration with Steve de Rosa (<a href="http://www.hvtn.org" target="_blank">HVTN</a>) we are working on utilizing high dimensional technologies that measure multiple functions (<em>e.g.</em> genes, cytokines) from individual cells to characterize cell poly-functionality in response to vaccination.
								 The working hypothesis is that cells producing multiple functions or certain combinations of functions are more beneficial to a clinical outcome such as control or prevention of infection.</dd>			  	    
						  	  </dl>
						  </div>
						  
						  <div class="tab-pane" id="5">
							  <p>For the past decade, microarrays have been the platform of choice for various high-throughput biological studies. In fact, our group has significantly contributed to the development of statistical methods for microarray data.
							 Recent improvements in the efficiency, quality, and cost of genome-wide sequencing are 
							 prompting biologists to rapidly abandon microarrays in favor of so-called next-generation sequencers. Next-generation sequencing technologies, capable of producing tens of 
							 millions of sequence reads during each instrument run, have already been applied for 
							 studying genome-wide transcription levels (RNA-Seq), transcription factor binding 
							 sites and nucleosome positioning (ChIP-Seq), and DNA methylation status. While 
							 sequencing-based studies have been promoted as overcoming longstanding limitations 
							 of microarray-based studies, these new biotechnologies raise similar as well as 
							 novel statistical and computational challenges. These sequencing technologies generate 
							 massive data sets (much larger than with microarrays), and there is a 
							 pressing need for statistical and computational tools to make sense of these data.
							  
							  To this end, our group has developed several methods for the analysis of ChIP-Seq and MNase-Seq data. The Figures below illustrate our analysis pipeline using 
					<a href="http://www.bioconductor.org/packages/devel/bioc/html/PICS.html" title="Probabilistic Inference for ChIP-Seq" target="_blank">PICS</a>,
					 <a title="de novo motif discovery" href="http://www.bioconductor.org/packages/devel/bioc/html/rGADEM.html" target="_blank">rGADEM</a>,
					 <a title="Motif Identification and Validation" href="http://www.bioconductor.org/packages/devel/bioc/html/MotIV.html" target="_blank">MotIV</a> and
					 <a href="http://www.bioconductor.org/packages/devel/bioc/html/PING.html" title="Probabilistic Inference for Nucleosome Positioning" target="_blank">PING</a>.
						</p><br/>
						<ul class="thumbnails">
						<li class="span6">
						 <a href="./utils/img/RG_SW_bwfigure2.png" class="thumbnail" id="epi1" title="ChIP-Seq analysis with PICS">
							<img src="./utils/img/RG_SW_bwfigure2.png" alt="PICS_example">
							</a>
							<h4>PICS increases spatial resolution of binding event detection</h4>
							<p>Transcription factor binding sites identified by PICS, iSeq, BayesPeak and MACS. PICS identifies two binding events in this region while other three methods identify these two events as one broad enriched region.</p>
							</li>
							<li class="span6">
							 <a href="./utils/img/RG_SW_bwfigure4.png" class="thumbnail" id="epi2" title="Nucleosome positioning analysis with PING">
								<img src="./utils/img/RG_SW_bwfigure4.png" alt="Nucleosome positioning analysis with PING">
								</a>
								<h4>PING identifies weakly positioned nucleosomes.</h4>
								<p>Expected Nucleosome positioning in Chr1 for 149000-153000 genomic region using PING and TpF. PING can identify nucleosome positioning with weak signal compared to TpF.</p>
								</li>							
							</ul>						  												
						  </div>
						  
						  <div class="tab-pane" id="6">
							  <p>Data resulting from novel high throughput technologies have lead to novel statistical 
							  problems and challenges. In addition, data are being accumulated more quickly than they can actually be analyzed. As a consequence, it is
							  essential that analytical tools and statistical methods evolve in
							  parallel with the assay technologies and datasets. Our group has developed many statistical methods for the analysis of high dimensional high throughput datasets including mixture models, sequential Monte Carlo, Markov chain Monte Carlo, and change point models (See examples below). 
							  We also implement state-of-the-art software for the analysis of large datasets using parallel programming (OpenMP, GPU) and efficient data formats (NetCDF, HDF5).
							  <ul class="thumbnails">
							  <li class="span6">
							   <a href="./utils/img/tiling-array-SMC.png" class="thumbnail" id="SC1" title="Change-point modeling of RNA tiling arrays">
							  	<img src="./utils/img/tiling-array-SMC.png" alt="Change-point">
							  	</a>
							  	<h4>Change-point modeling of DNA tiling arrays</h4>
							  	<p>Segmentation results for part of chromosome 1 for the yeast genome expression data using our SMC algorithm and least-squares change point model estimation. For the top graph, the MAP estimate is displayed with transcript segments (green background), non-transcript segments (white background) segments and black segments for the segment intensity levels. For both top and bottom graphs, segments boundaries are represented with green vertical lines. Transcript annotations are shown below with red rectangles representing coding sequences and black segments representing TF binding sites. More precise estimation of known transcript boundaries can be obtained with <a href="http://www.springerlink.com/content/3880054343602gw8/" target="_BLANK">our approach</a>.</p>
							  	</li>
							  	<li class="span6">
							  	 <a href="./utils/img/SMC-MCMC.png" class="thumbnail" id="SC2" title="Regularization path plots">
							  		<img src="./utils/img/SMC-MCMC.png" alt="Regularization-path-SMC">
							  		</a>
							  		<h4>Lasso-type regularization path plots SMC <em>vs.</em> MCMC</h4>
							  		<p>Plots using MCMC and SMC for fixed computational time of 5 min. The MCMC plot 
							  		is a lot more noisy than the SMC version (<a href="http://www.springerlink.com/content/3880054343602gw8/"target="_BLANK">our approach</a>). </p>
							  		</li>							
							  	</ul>						  												
							  
							  </p>
						  </div>
					  </div>
				  </div> <!-- /tabbable -->			  
			  </div><!-- /span -->
		  </div><!-- /row -->
	  </div><!-- /container -->		  
		
	  <div class="container">
		  <div class="row">																
			  <div class="span12">
				  <!-- Footer ================================================== -->
				  <footer class="footer">
					  <p class="pull-right"><a href="#">Back to top</a></p>
					  <p class="pull-left">
						  <a href="https://twitter.com/raphg" class="twitter-follow-button" data-width="225px" data-link-color="#0069D6" data-show-count="true">Follow @raphg</a>
						  <a href="https://twitter.com/share" class="twitter-share-button" data-url="http://twitter.github.com/raphg/" data-count="horizontal" data-via="raphg" >Tweet</a></p>
						  <br /><br />
						  <p class="pull-left"> &copy; 2012 <a href="#">rglab</a>. All right reserved.</p>
						  <br />
						  <p class="pull-left">Made with <a href="http://twitter.github.com/bootstrap">Bootstrap</a>.</p>
						  
					  </footer>
			  </div><!--span12-->
		  </div>
	  </div><!-- /container -->
    
    <!-- Le javascript -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script type="text/javascript" src="http://platform.twitter.com/widgets.js"></script>	
    <script src="./utils/js/jquery.js"></script>
    <script src="./utils/js/bootstrap.js"></script>
	<script type="text/javascript" src="./utils/js/jquery.tweetable.js"></script>

	  <!-- Add fancyBox -->
	<link rel="stylesheet" href="./utils/fancybox/source/jquery.fancybox.css?v=2.1.0" type="text/css" media="screen" />
	<script type="text/javascript" src="./utils/fancybox/source/jquery.fancybox.pack.js?v=2.1.0"></script>

	
	<script type="text/javascript">
	$(document).ready(function() {
	$("#cyto1").fancybox({
	helpers : {
		title : {
			type : 'over'
		}
	}
	});});
	
	$(document).ready(function() {
	$("#flow1").fancybox({
	helpers : {
		title : {
			type : 'over'
		}
	}
	});});
	
	$(document).ready(function() {
	$("#cyto2").fancybox({
	helpers : {
		title : {
			type : 'over'
		}
	}
	});});
	$(document).ready(function() {
	$("#pep1").fancybox({
    	helpers : {
    		title : {
    			type : 'over'
    		}
    	}
    });	});
    
    $(document).ready(function() {
    $("#pep2").fancybox({
    	helpers : {
    		title : {
    			type : 'over'
    		}
    	}
    });	});

	$(document).ready(function() {
	$("#epi1").fancybox({
		helpers : {
			title : {
				type : 'over'
			}
		}
	});	});

	$(document).ready(function() {
	$("#epi2").fancybox({
		helpers : {
			title : {
				type : 'over'
			}
		}
	});	});

	$(document).ready(function() {
	$("#SC1").fancybox({
		helpers : {
			title : {
				type : 'over'
			}
		}
	});	});

	$(document).ready(function() {
	$("#SC2").fancybox({
		helpers : {
			title : {
				type : 'over'
			}
		}
	});	});
    
	</script>	
	
  </body>
</html>
